opentelemetry-sdk/src/metrics/pipeline.rs

```diff
@@ -385,7 +386,11 @@ where
                 .clone()
                 .map(|allowed| Arc::new(move |kv: &KeyValue| allowed.contains(&kv.key)) as Arc<_>);
 
-            let b = AggregateBuilder::new(self.pipeline.reader.temporality(kind), filter);
+            let b = AggregateBuilder::new(
+                self.pipeline.reader.temporality(kind),
+                filter,
+                stream.cardinality_limit.unwrap_or(2000),
```

Comment:
Do we have any logging for instrument creation? We should add some information about the cardinality limit to the logs.

Reply:
added cardinality info to the existing log.